src/locale/en-US.properties

```diff
@@ -4,3 +4,5 @@ enableAlphabeticSort_title = Enable alphabetic sorting
 add_group = Create new group
 panelButton_label = Group Tabs
 unnamed_group = Unnamed Group
+close_group_prompt_title = Close group
```

Comment:
Just for the sake of easier translation, I try to keep the lower block of this file ordered alphabetically. Could you move your two strings below `add_group`?